tests/integrationtest/r/ddl/active_active.result

```diff
@@ -43,6 +43,20 @@ t3	CREATE TABLE `t3` (
   PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
 create table t4 (id int primary key, name varchar(50)) /*T![active_active] ACTIVE_ACTIVE='ON' */ /*T![softdelete] SOFTDELETE=RETENTION 7 DAY */ /*T![softdelete] SOFTDELETE_JOB_ENABLE='ON' */ /*T![softdelete] SOFTDELETE_JOB_INTERVAL='24h' */;
+ALTER TABLE t4 ADD INDEX i1(_tidb_origin_ts);
+ALTER TABLE t4 ADD INDEX i2(_tidb_softdelete_time);
+ALTER TABLE t4 ADD UNIQUE INDEX i3(_tidb_softdelete_time);
+Error 1846 (0A000): UNIQUE is not supported. Reason: Cannot add unique index on ACTIVE_ACTIVE or softdelete table. Try removing UNIQUE.
+ALTER TABLE t4 ADD UNIQUE INDEX i3(_tidb_origin_ts);
+Error 1846 (0A000): UNIQUE is not supported. Reason: Cannot add unique index on ACTIVE_ACTIVE or softdelete table. Try removing UNIQUE.
+ALTER TABLE t4 ADD UNIQUE INDEX i3(_tidb_commit_ts);
+Error 1846 (0A000): UNIQUE is not supported. Reason: Cannot add unique index on ACTIVE_ACTIVE or softdelete table. Try removing UNIQUE.
+ALTER TABLE t4 ADD UNIQUE INDEX i3(name);
+Error 1846 (0A000): UNIQUE is not supported. Reason: Cannot add unique index on ACTIVE_ACTIVE or softdelete table. Try removing UNIQUE.
+ALTER TABLE t4 ADD INDEX i3(_tidb_commit_ts);
+Error 1072 (42000): column does not exist: _tidb_commit_ts
+ALTER TABLE t4 ADD CONSTRAINT UNIQUE(_tidb_commit_ts);
```

Comment:
Also need to forbid CONSTRAINT CHECK for any _tidb_xxx columns:

https://docs.pingcap.com/tidb/stable/constraints/#check